packages/hadron-build/lib/target.js

```diff
@@ -680,7 +680,12 @@ class Target {
         const createDeb = require('electron-installer-debian');
         debug('creating deb...', this.installerOptions.deb);
         return createDeb(this.installerOptions.deb).then(() => {
-          return sign(this.dest(this.linux_deb_filename));
+          // We do not sign debs because it doesn't work, see
```

Comment:
I think we can delete this comment, the slack thread is going to be deleted in 30 days

Reply:
Oh, I thought it's only images that are deleted in 30 days... okay, will clean it up